app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -122,7 +122,7 @@ abstract class RoutePresenter(
         val layoutInflater = LayoutInflater.from(activity)
 
         toast.view = layoutInflater.inflate(R.layout.toast_view, container, false)
-        toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, 216)
+        toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, action.viewModel.bottomMargin)
```

Comment:
I don't think this is the correct approach. The position of the toast is a function of the screen that it's on, rather than the actual toast that needs to be displayed.

The margin needs to be chosen in the `RoutePresenter`.

There are two other approaches I can think of: 

1. Easy: choose the bottom margin  based on whether or not the current destination is the `ItemList`.
2. Harder: allow the bottom margin to be determined from something in the layout of the current fragment or activity.